source/django_package/domain/import_datas/parsing_config_file.py

```diff
@@ -0,0 +1,23 @@
+import yaml
+
+
+class ReadConfigFile:
+    config_file_keys = []
+    config_file_values = []
+    data_source = []
+    data_type = []
+
+    def parse_file(self):
+        with open('csv_config_file.yaml') as f:
+            docs = yaml.load_all(f, Loader=yaml.FullLoader)
+
+            for doc in docs:
+                for k, v in doc.items():
+                    self.config_file_keys.append(k)
+                    self.config_file_values.append(v)
```

Comment:
I don't think the structure of the config file is going to be completely flat. Perhaps you don't need all this manipulation of keys and values; just return whatever pyyaml is producing. The resulting method will be much simpler. On the other hand, in the future we might implement other stuff here, like verifying the syntax of the file.